libr/core/project.c

```diff
@@ -176,9 +176,24 @@ R_API int r_core_project_delete(RCore *core, const char *prjfile) {
 	}
 	if (r_core_is_project (core, prjfile)) {
 		// rm project file
-		r_file_rm (path);
-		eprintf ("rm %s\n", path);
-		path = r_str_append (path, ".d");
+		if(r_file_exists (path)) {
+			r_file_rm (path);
+			eprintf ("rm %s\n", path);
+		}
+
+		//rm xrefs.sdb file
+		path = r_file_dirname (path);
+		path = r_str_append (path, R_SYS_DIR);
+		path = r_str_append (path, "xrefs.sdb");
+		if(r_file_exists (path)) {
+			r_file_rm (path);
+			eprintf ("rm %s\n", path);
+		}
+		//change path to rop.d
+		path = r_file_dirname (path);
```

Comment:
Memory leak

Reply:
Why would there be memory leak? `path` variable is freed at the end of the function. Am I missing something?